src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -277,7 +280,10 @@ class QueryAppsDataValidator {
         LOG4CXX_WARN(logger_,
                      kQueryAppsValidationFailedPrefix
                          << "'languages.ttsName' doesn't exist");
-        return false;
+        if(app_id.empty())
+            return false;
+        language[language_name][json::ttsName] = app_id;
```

Comment:
@pold500 confusing situation that function ValidateLangiages change something in languages.